khi_robot_control/src/khi_robot_krnx_driver.cpp

```diff
@@ -607,19 +609,19 @@ bool KhiRobotKrnxDriver::writeData( const int cont_no, JointData joint )
         /* ERROR log */
         for ( int ano = 0; ano < robot_info[cont_no].arm_num; ano++ )
         {
-            snprintf( msg, sizeof(msg), "[krnx_PrimeRtcCompData] ano:%d [jt]status:diff ", ano+1 );
+            snprintf( msg, sizeof(msg), "[krnx_PrimeRtcCompData] ano:%d [jt]pos:diff:status ", ano+1 );
             krnx_GetRtcCompData( cont_no, ano, &rtc_old_comp[cont_no][ano][0] );
-            for ( int cnt = 0; cnt < p_rb_tbl[cont_no]->arm_tbl[ano].jt_num; cnt++ )
+            getCurMotionData( cont_no, ano, &data );
+            for ( int jt = 0; jt < p_rb_tbl[cont_no]->arm_tbl[ano].jt_num; jt++ )
             {
-                if ( p_rb_tbl[cont_no]->arm_tbl[ano].jt_tbl[cnt].type == TYPE_LINE )
-                {
-                    diff = ( rtc_comp[cont_no][ano][cnt] - rtc_old_comp[cont_no][ano][cnt] )/KHI_KRNX_M2MM*(1e+9/robot_info[cont_no].period);
-                }
-                else
+                jt_pos = data.ang_ref[jt];
+                rtc_diff = ( rtc_comp[cont_no][ano][jt] - rtc_old_comp[cont_no][ano][jt] )*(1e+9/robot_info[cont_no].period);
+                if ( p_rb_tbl[cont_no]->arm_tbl[ano].jt_tbl[jt].type == TYPE_RAD )
```

Comment:
In above docs ,` |diff|float|joint command difference value[deg or mm] between period|`
I think it is need to change [m] to [mm].

Reply:
`rtc_comp`  is   already [mm] units in this line,is it?
I had been misunderstanding.

But ROS standard units is SI units.
SO these error messages had better to been also displayed in [rad] or [m].

Reply:
OK, I changed units about error messages.